components/scream/src/physics/shoc/atmosphere_macrophysics.hpp

```diff
@@ -11,11 +11,11 @@ namespace scream
 
 /*
  * The class responsible to handle the atmosphere microphysics
- * 
+ *
  * The AD should store exactly ONE instance of this class stored
  * in its list of subcomponents (the AD should make sure of this).
- * 
- *  Note: for now, scream is only going to accommodate SHOC and as macrophysics 
+ *
+ *  Note: for now, scream is only going to accommodate SHOC and as macrophysics
```

Comment:
Tiny nitpick: SHOC and what? Can we strike the "and" here?

Reply:
right, will fix that comment